docs/csharp/index.yml

```
@@ -0,0 +1,113 @@
+### YamlMime:Landing
+
+title: "C# documentation"
+summary: "Learn how to write any application using the C# programming language on the .NET platform."
+
+metadata:
+  title: "C# docs - get started, tutorials, reference."
+  description: "Learn C# programming - for beginning developers, developers new to C#, and experienced C# / .NET developers"
+  ms.topic: landing-page # Required
+  author: billwagner
+  ms.author: wiwagn
+  ms.date: 11/19/2019
+
+landingContent:
+  - title: "Learn to program in C#"
+    linkLists:
+      - linkListType: get-started
+        links:
+          - text: Online, interactive Hello world
+            url: tutorials/intro-to-csharp/hello-world.yml
+          - text: "Math and numbers in C#"
+            url: tutorials/intro-to-csharp/numbers-in-csharp.yml
+          - text: Introductory tutorials
+            url: tutorials/intro-to-csharp/index.md
+          - text: "Create C# apps with Visual Studio"
+            url: https://docs.microsoft.com/visualstudio/get-started/csharp/?view=vs-2019
```

To make the link goes to the latest version

```suggestion
            url: https://docs.microsoft.com/visualstudio/get-started/csharp/
```

---

Actually since landing pages aren't supposed to link to other landing pages, how about linking here instead? /visualstudio/get-started/csharp/tutorial-console